[2017-02-23 18:11:57.030] [ERROR] netWork - 写入session失败 { SequelizeBaseError: Validation error
    at Query.formatError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:196:16)
    at Query.AbstractQuery.run.Utils.Promise.connection.query [as onResult] (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:55:23)
    at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:31:12)
    at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
    at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
    at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
    at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
  name: 'SequelizeUniqueConstraintError',
  errors: 
   [ ValidationErrorItem {
       message: 'PRIMARY must be unique',
       type: 'unique violation',
       path: 'PRIMARY',
       value: '20000' } ],
  fields: { PRIMARY: '20000' },
  parent: 
   { Error: Duplicate entry '20000' for key 'PRIMARY'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '#23000',
     sql: 'INSERT INTO `user_session` (`id`,`user_id`,`token`,`login_location`) VALUES (DEFAULT,10000,\'6e4bc18c-22e4-4246-8098-e548845d1040\',\'北京市\');' },
  original: 
   { Error: Duplicate entry '20000' for key 'PRIMARY'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '#23000',
     sql: 'INSERT INTO `user_session` (`id`,`user_id`,`token`,`login_location`) VALUES (DEFAULT,10000,\'6e4bc18c-22e4-4246-8098-e548845d1040\',\'北京市\');' },
  sql: 'INSERT INTO `user_session` (`id`,`user_id`,`token`,`login_location`) VALUES (DEFAULT,10000,\'6e4bc18c-22e4-4246-8098-e548845d1040\',\'北京市\');' }
[2017-02-23 18:12:37.532] [ERROR] UserService - 写入session失败 { SequelizeBaseError: Validation error
    at Query.formatError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:196:16)
    at Query.AbstractQuery.run.Utils.Promise.connection.query [as onResult] (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:55:23)
    at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:31:12)
    at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
    at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
    at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
    at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
  name: 'SequelizeUniqueConstraintError',
  errors: 
   [ ValidationErrorItem {
       message: 'PRIMARY must be unique',
       type: 'unique violation',
       path: 'PRIMARY',
       value: '20000' } ],
  fields: { PRIMARY: '20000' },
  parent: 
   { Error: Duplicate entry '20000' for key 'PRIMARY'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '#23000',
     sql: 'INSERT INTO `user_session` (`id`,`user_id`,`token`,`login_location`) VALUES (DEFAULT,10000,\'c7a2dea4-7766-476f-9ca0-aa61ed0295ee\',\'北京市\');' },
  original: 
   { Error: Duplicate entry '20000' for key 'PRIMARY'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_DUP_ENTRY',
     errno: 1062,
     sqlState: '#23000',
     sql: 'INSERT INTO `user_session` (`id`,`user_id`,`token`,`login_location`) VALUES (DEFAULT,10000,\'c7a2dea4-7766-476f-9ca0-aa61ed0295ee\',\'北京市\');' },
  sql: 'INSERT INTO `user_session` (`id`,`user_id`,`token`,`login_location`) VALUES (DEFAULT,10000,\'c7a2dea4-7766-476f-9ca0-aa61ed0295ee\',\'北京市\');' }
[2017-02-23 18:28:09.630] [ERROR] UserService - 写入session失败 ReferenceError: data is not defined
    at logout (/Users/ruanxiaoquan/project/qingqujun-server/service/account/UserService.js:76:22)
    at self.userLogin.db.User.findOne.then.db.UserSession.create.then (/Users/ruanxiaoquan/project/qingqujun-server/service/account/UserService.js:53:17)
    at tryCatcher (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/util.js:16:23)
    at Promise.module.exports.Promise._settlePromiseFromHandler (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise.module.exports.Promise._settlePromise (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise.module.exports.Promise._settlePromise0 (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise.module.exports.Promise._settlePromises (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.__dirname.Async.drainQueues (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:651:20)
    at tryOnImmediate (timers.js:624:5)
    at processImmediate [as _immediateCallback] (timers.js:596:5)
[2017-02-23 18:29:23.769] [ERROR] UserService - 写入session失败 Error: Missing where attribute in the options parameter passed to update.
    at Function.update (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/model.js:2494:13)
    at logout (/Users/ruanxiaoquan/project/qingqujun-server/service/account/UserService.js:71:20)
    at self.userLogin.db.User.findOne.then.db.UserSession.create.then (/Users/ruanxiaoquan/project/qingqujun-server/service/account/UserService.js:50:17)
    at tryCatcher (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/util.js:16:23)
    at Promise.module.exports.Promise._settlePromiseFromHandler (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise.module.exports.Promise._settlePromise (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise.module.exports.Promise._settlePromise0 (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise.module.exports.Promise._settlePromises (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.__dirname.Async.drainQueues (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:651:20)
    at tryOnImmediate (timers.js:624:5)
    at processImmediate [as _immediateCallback] (timers.js:596:5)
[2017-02-23 19:01:42.019] [ERROR] UserService - 登录时查询用户信息失败 Error: Invalid value [object Object]
    at Object.escape (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/sql-string.js:59:11)
    at Object.escape (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:949:22)
    at Object.whereItemQuery (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2206:22)
    at QueryGenerator.whereItemsQuery._.forOwn (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1857:25)
    at isBind (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/node_modules/lodash/lodash.js:4738:15)
    at baseForOwn (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/node_modules/lodash/lodash.js:2879:24)
    at Function.forOwn (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/node_modules/lodash/lodash.js:12670:24)
    at Object.whereItemsQuery (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1856:9)
    at Object.getWhereConditions (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2270:19)
    at Object.selectQuery (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1414:28)
    at QueryInterface.select (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/query-interface.js:634:27)
    at __dirname.static.findAll.Promise.try.then.then.then (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/model.js:1484:34)
    at tryCatcher (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/util.js:16:23)
    at Promise.module.exports.Promise._settlePromiseFromHandler (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise.module.exports.Promise._settlePromise (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise.module.exports.Promise._settlePromise0 (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise.module.exports.Promise._settlePromises (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.__dirname.Async.drainQueues (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:651:20)
    at tryOnImmediate (timers.js:624:5)
[2017-02-23 19:04:03.214] [ERROR] UserService - 登录时查询用户信息失败 Error: Invalid value [object Object]
    at Object.escape (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/sql-string.js:59:11)
    at Object.escape (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:949:22)
    at Object.whereItemQuery (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2206:22)
    at _.forOwn (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1857:25)
    at isBind (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/node_modules/lodash/lodash.js:4738:15)
    at baseForOwn (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/node_modules/lodash/lodash.js:2879:24)
    at Function.forOwn (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/node_modules/lodash/lodash.js:12670:24)
    at Object.whereItemsQuery (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1856:9)
    at Object.getWhereConditions (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2270:19)
    at Object.selectQuery (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1414:28)
    at QueryInterface.select (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/query-interface.js:634:27)
    at Promise.try.then.then.then (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/model.js:1484:34)
    at tryCatcher (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/util.js:16:23)
    at Promise.module.exports.Promise._settlePromiseFromHandler (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise.module.exports.Promise._settlePromise (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise.module.exports.Promise._settlePromise0 (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise.module.exports.Promise._settlePromises (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.__dirname.Async.drainQueues (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:651:20)
    at tryOnImmediate (timers.js:624:5)
[2017-02-23 19:06:39.186] [ERROR] UserService - 登录时查询用户信息失败 Error: Invalid value [object Object]
    at Object.escape (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/sql-string.js:59:11)
    at Object.escape (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:949:22)
    at Object.whereItemQuery (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2206:22)
    at _.forOwn (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1857:25)
    at isBind (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/node_modules/lodash/lodash.js:4738:15)
    at baseForOwn (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/node_modules/lodash/lodash.js:2879:24)
    at Function.forOwn (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/node_modules/lodash/lodash.js:12670:24)
    at Object.whereItemsQuery (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1856:9)
    at Object.getWhereConditions (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2270:19)
    at Object.selectQuery (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1414:28)
    at QueryInterface.select (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/query-interface.js:634:27)
    at Promise.try.then.then.then (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/model.js:1484:34)
    at tryCatcher (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/util.js:16:23)
    at Promise.module.exports.Promise._settlePromiseFromHandler (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise.module.exports.Promise._settlePromise (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise.module.exports.Promise._settlePromise0 (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise.module.exports.Promise._settlePromises (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.__dirname.Async.drainQueues (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:651:20)
    at tryOnImmediate (timers.js:624:5)
[2017-02-23 19:09:53.326] [ERROR] UserService - 登录时查询用户信息失败 { SequelizeBaseError: Unknown column 'id' in 'field list'
    at Query.formatError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:217:16)
    at Query.Utils.Promise.connection.query [as onResult] (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:55:23)
    at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:31:12)
    at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
    at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
    at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
    at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
  name: 'SequelizeDatabaseError',
  parent: 
   { Error: Unknown column 'id' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' },
  original: 
   { Error: Unknown column 'id' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' },
  sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' }
[2017-02-23 19:12:03.658] [ERROR] UserService - 登录时查询用户信息失败 { SequelizeBaseError: Unknown column 'id' in 'field list'
    at Query.formatError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:217:16)
    at Query.Utils.Promise.connection.query [as onResult] (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:55:23)
    at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:31:12)
    at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
    at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
    at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
    at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
  name: 'SequelizeDatabaseError',
  parent: 
   { Error: Unknown column 'id' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' },
  original: 
   { Error: Unknown column 'id' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' },
  sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' }
[2017-02-23 19:13:42.607] [ERROR] UserService - 登录时查询用户信息失败 { SequelizeBaseError: Unknown column 'id' in 'field list'
    at Query.formatError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:217:16)
    at Query.Utils.Promise.connection.query [as onResult] (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:55:23)
    at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:31:12)
    at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
    at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
    at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
    at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
  name: 'SequelizeDatabaseError',
  parent: 
   { Error: Unknown column 'id' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' },
  original: 
   { Error: Unknown column 'id' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' },
  sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' }
[2017-02-23 19:17:01.304] [ERROR] UserService - 登录时查询用户信息失败 { SequelizeBaseError: Unknown column 'id' in 'field list'
    at Query.formatError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:217:16)
    at Query.AbstractQuery.run.Utils.Promise.connection.query [as onResult] (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:55:23)
    at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:31:12)
    at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
    at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
    at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
    at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
  name: 'SequelizeDatabaseError',
  parent: 
   { Error: Unknown column 'id' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' },
  original: 
   { Error: Unknown column 'id' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' },
  sql: 'SELECT `id`, `user_id`, `nick_name`, `birthday`, `age`, `photo`, `phone`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` WHERE `VUserInfo`.`user_id` = 10000 LIMIT 1;' }
[2017-02-23 19:25:11.558] [ERROR] UserService - 登录时查询用户信息失败 { SequelizeBaseError: Unknown column 'register_location' in 'field list'
    at Query.formatError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:217:16)
    at Query.AbstractQuery.run.Utils.Promise.connection.query [as onResult] (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:55:23)
    at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:31:12)
    at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
    at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
    at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
    at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
  name: 'SequelizeDatabaseError',
  parent: 
   { Error: Unknown column 'register_location' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' },
  original: 
   { Error: Unknown column 'register_location' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' },
  sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' }
[2017-02-23 19:25:52.565] [ERROR] UserService - 登录时查询用户信息失败 { SequelizeBaseError: Unknown column 'register_location' in 'field list'
    at Query.formatError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:217:16)
    at Query.AbstractQuery.run.Utils.Promise.connection.query [as onResult] (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:55:23)
    at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:31:12)
    at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
    at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
    at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
    at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
  name: 'SequelizeDatabaseError',
  parent: 
   { Error: Unknown column 'register_location' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' },
  original: 
   { Error: Unknown column 'register_location' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' },
  sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' }
[2017-02-23 19:26:21.399] [ERROR] UserService - 登录时查询用户信息失败 { SequelizeBaseError: Unknown column 'register_location' in 'field list'
    at Query.formatError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:217:16)
    at Query.AbstractQuery.run.Utils.Promise.connection.query [as onResult] (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:55:23)
    at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:31:12)
    at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
    at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
    at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
    at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
  name: 'SequelizeDatabaseError',
  parent: 
   { Error: Unknown column 'register_location' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' },
  original: 
   { Error: Unknown column 'register_location' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' },
  sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' }
[2017-02-23 19:28:13.332] [ERROR] UserService - 登录时查询用户信息失败 { SequelizeBaseError: Unknown column 'register_location' in 'field list'
    at Query.formatError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:217:16)
    at Query.AbstractQuery.run.Utils.Promise.connection.query [as onResult] (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/sequelize/lib/dialects/mysql/query.js:55:23)
    at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:31:12)
    at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
    at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
    at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
    at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
  name: 'SequelizeDatabaseError',
  parent: 
   { Error: Unknown column 'register_location' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' },
  original: 
   { Error: Unknown column 'register_location' in 'field list'
       at Packet.asError (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packets/packet.js:667:13)
       at Query.Command.execute (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/commands/command.js:29:22)
       at Connection.handlePacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:417:28)
       at PacketParser.onPacket (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:93:16)
       at PacketParser.executeStart (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/packet_parser.js:73:14)
       at Socket.<anonymous> (/Users/ruanxiaoquan/project/qingqujun-server/node_modules/mysql2/lib/connection.js:101:29)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:189:7)
       at readableAddChunk (_stream_readable.js:176:18)
       at Socket.Readable.push (_stream_readable.js:134:10)
       at TCP.onread (net.js:551:20)
     code: 'ER_BAD_FIELD_ERROR',
     errno: 1054,
     sqlState: '#42S22',
     sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' },
  sql: 'SELECT `user_id`, `nick_name`, `birthday`, `age`, `photo`, `is_lock`, `loaction`, `signature`, `register_location`, `create_time` FROM `v_user_info` AS `VUserInfo` LIMIT 1;' }
